container-toolkit/supported-platforms.md

```diff
@@ -16,7 +16,12 @@ backlinks: none
 
 ## Linux Distributions
 
-Supported Linux distributions are listed below:
+The NVIDIA Container Toolkit has been qualified against the following platforms.
```

Comment:
I suspect your choice of "qualified" rather than supported is to communicate some subtlety.  If feels dicey to me, but it's you who answers questions and not me.

nit: s/has been/is/, unless there is a consideration that I'm overlooking.

Reply:
The issue is that people reach out to us all the time pointing out that we need to add "Debian 12" support or "Ubuntu 24.04" support, where we are reasonably confident that the packages should work as is. The caveat is that we have not officiallty tested against these configurations. If you have a suggestion on how to better call this out, that would be appreciated.

Reply:
For example: https://github.com/NVIDIA/nvidia-container-toolkit/issues/482#issuecomment-2122204585